pyproject.toml

```diff
@@ -108,6 +108,16 @@ visual_prompt = [
     "scikit-image",
 ]
 
+detection = [
+    "mmpretrain==1.0.*",
+    "mmdet==3.1.0",
+    "mmdeploy",
+    "timm==0.6.12",
+    "pytorchcv",
+    "yapf<0.40.0",
```

Comment:
I'm not sure, but OTX1.x need yapf for all tasks. In OTXv2 there could be a chance for removing this. 
Since classification already require yapf, how about deleting this after phase 1 and validate with our integration tests